core/domain/subscription_services.py

```diff
@@ -17,7 +17,6 @@
 """Services for managing subscriptions."""
 
 import datetime
-
```

Comment:
Keep the blank line. It separates Python stdlib imports from the others.

Reply:
Done!